x-pack/plugins/security/server/authentication/providers/pki.test.ts

```diff
@@ -399,55 +408,102 @@ describe('PKIAuthenticationProvider', () => {
           LegacyElasticsearchErrorHelpers.decorateNotAuthorizedError(new Error())
         )
         // In response to a call with a new token.
+        .mockResolvedValueOnce(user) // In response to call with an expired token.
```

Comment:
__note:__ I'll remove this stupid setup in the scope of https://github.com/elastic/kibana/issues/80952 soon.